src/GitHubCreateMergePRs/Program.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        // Default when no args passed in.
+        var isDryRun = true;
+        var isAutomated = false;
+        var githubToken = string.Empty;
+
+        foreach (var arg in args)
+        {
+            if (arg.StartsWith("--isDryRun"))
+            {
+                isDryRun = bool.Parse(GetArgumentValue(arg));
+            }
+            else if (arg.StartsWith("--isAutomated"))
+            {
+                isAutomated = bool.Parse(GetArgumentValue(arg));
+            }
+            else if (arg.StartsWith("--githubToken"))
+            {
+                githubToken = GetArgumentValue(arg);
+            }
+        }
+
+        Console.WriteLine($"Executing with {nameof(isDryRun)}={isDryRun}, {nameof(isAutomated)}={isAutomated}, {nameof(githubToken)}={githubToken}");
+        var config = GetConfig();
+        RunAsync(config, isAutomated, isDryRun, githubToken).GetAwaiter().GetResult();
```

Comment:
Why not just have `async Main` here so we can properly `await` this call?

Reply:
That makes sense!  Will address in followup